src/library/scala/collection/convert/LongAccumulator.scala

```diff
@@ -58,7 +75,7 @@ final class LongAccumulator
   override def result(): LongAccumulator = this
 
   /** Removes all elements from `that` and appends them to this `LongAccumulator`. */
-  final def drain(that: LongAccumulator): Unit = {
+  def drain(that: LongAccumulator): Unit = {
```

Comment:
I'm not sure why you removed `final` from this method?  It relies upon hidden (package private) implementation details and thus is quite difficult to override correctly.

Reply:
Because its class (`LongAccumulator`) is final, so the keyword is redundant

Reply:
(Or do you think the accumulator implementation classes should be non-final?)

Reply:
Oops!  Missed that.  Never mind then!  Leaving the classes final is good; they're hard to inherit from productively because of the fiddly internals.